docs/promise.md

```diff
@@ -13,18 +13,18 @@ function loadJSONSync(filename: string) {
     return JSON.parse(fs.readFileSync(filename));
 }
 
-// good json file
+// geçerli json dosyası
 console.log(loadJSONSync('good.json'));
 
-// non-existent file, so fs.readFileSync fails
+// var olmayan json dosyası. bu yüzden fs.readFilesync hata verir
 try {
     console.log(loadJSONSync('absent.json'));
 }
 catch (err) {
     console.log('absent.json error', err.message);
 }
 
-// invalid json file i.e. the file exists but contains invalid JSON so JSON.parse fails
+// geçersiz json dosyası. dosya var ama içindeki JSON geçersiz. bu yüzden JSON.parse hata verir
```

Comment:
Comment'lerde büyük harf olmasına pek alışık değilim. Noktadan sonrakileri büyük yaparsak comment'in ilk harfi de büyük olmalı. Bunu tartışalım mı yoksa tartışmaya değmeyecek bir konu mu bilmiyorum ama şimdilik senin dediğin gibi yapıcam.